selfdrive/locationd/locationd.cc

```diff
@@ -521,10 +579,15 @@ int Localizer::locationd_thread() {
 
   uint64_t cnt = 0;
   bool filterInitialized = false;
+  const std::vector<std::string> critical_input_services = {"cameraOdometry", "liveCalibration", "accelerometer", "gyroscope"};
```

Comment:
does it make more sense to blacklist non-critical services? that way new services will be checked by default